examples/how_to/use_crumbs.py

```diff
@@ -12,4 +12,4 @@
                   ))
 
 for job_name in jenkins.jobs:
-    print job_name
+    print (job_name)
```

Comment:
Why do this?

Reply:
Thanks so much for merging!  I only did it because I started bringing a fork in house in our sonar scan was complaining that this was a blocker issue.  When I looked into it further I found that the print statement was deprecated in Python 3, and that the function should be used going forward.

https://docs.python.org/3/whatsnew/3.0.html